modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -60,12 +139,17 @@ metadata:
 spec:
   machineConfigPoolSelector:
     matchLabels:
-      custom-kubelet: large-pods
+      custom-kubelet: set-max-pods <1>
   kubeletConfig:
-    maxPods: 500
+    maxPods: 500 <2>
 ----
+<1> Enter the label from the Machine Config Pool.
```

Comment:
Looks like this is the only init cap instance, so:

s/Machine Config Pool/machine config pool/